packages/stacks-docs/product/components/check.html

```diff
@@ -0,0 +1,557 @@
+                        <label class="s-label" for="state-radio-{{ item.label }}">
+                            {{ item.label }}
+                            <p class="s-description">{{ item.description }}</p>
+                        </label>
+                    </div>
+                {% endfor %}
+            </fieldset>
+
+            <!-- Checkmark w/ radio using .s-menu for additional styling -->
+            <div class="s-popover is-visible ps-relative ws2 p8">
+                <fieldset class="s-menu s-check-group">
+                    <legend class="s-menu--title">Pick a fruit</legend>
+                    {% for item in check.group-items %}
+                        <div class="s-menu--item">
+                            <label class="s-menu--action s-check s-check__checkmark has-{{ item.state }}" for="state-checkmark-{{ item.label }}">
+                                <input type="radio" name="state-checkmark" id="state-checkmark-{{ item.label }}" {% if forloop.first %} checked {% endif %} />
+                                <div>
+                                    {{ item.label }}
+                                    <p class="s-description">{{ item.description }}</p>
+                                </div>
+                            </label>
+                        </div>
+                    {% endfor %}
+                </fieldset>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="stacks-section">
+    {% header "h2", "Indeterminate state" %}
+    <p class="stacks-copy">Checkboxes can be styled by using the <code>:indeterminate</code> pseudo class.</p>
+    {% tip, "", "mb32" %}
+        <strong>Note:</strong> The <code>:indeterminate</code> pseudo class can only be set via JavaScript. Use the <code>HTMLInputElement</code> object's <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes"><code>indeterminate</code></a> property to set the state.
+    {% endtip %}
+
+    <div class="stacks-preview">
+{% highlight html %}
+<fieldset class="s-check-group">
+    <div class="s-check">
+        <input type="checkbox" name="" id="indeterminate-checkbox-1" />
+        <label class="s-label" for="indeterminate-checkbox-1">Select all</label>
+    </div>
+</fieldset>
+
+<script>
+    document.getElementById("indeterminate-checkbox-1").indeterminate = true;
+</script>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <fieldset class="s-check-group">
+                <div class="s-check">
+                    <input type="checkbox" name="" id="indeterminate-checkbox-1" />
+                    <label class="s-label" for="indeterminate-checkbox-1">Select all</label>
+                </div>
+            </fieldset>
+        </div>
+    </div>
+</section>
+
+<script>
+    document.getElementById("indeterminate-checkbox-1").indeterminate = true;
+</script>
```

Comment:
Should we also add an onclick handler to toggle the indeterminate state? All the other examples on the page work when you click on them besides this one.